### DISCUSSION TITLE
Stream music from my Plex Server?

### DISCUSSION THREAD

**Discussion by NostalgicStone** [jagrosh/MusicBot#1179] [Questions]
I have the bot set up and working, tested with YouTube and local media.
Can I set it to stream from my own Plex Server?

I tried creating playlist in plex, opening in web browser and copying the URL to us in the PLAY command, but it does nothing.
Thanks in advance

**Comment by MichailiK** [comment]
Plex is currently not part of the [support sources](https://github.com/jagrosh/MusicBot#supported-sources-and-formats).

As an alternative, you could try using sshfs to mount to your plex server, then tell JMusicBot to play a local file.
